ios/Video/RCTVideo.swift

```diff
@@ -1045,7 +1045,9 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     func createPlayerViewController(player: AVPlayer, withPlayerItem _: AVPlayerItem) -> RCTVideoPlayerViewController {
         let viewController = RCTVideoPlayerViewController()
         viewController.showsPlaybackControls = self._controls
-        viewController.updatesNowPlayingInfoCenter = false
+        #if !os(tvOS)
+            viewController.updatesNowPlayingInfoCenter = false
+        #endif
```

Comment:
I don't like it... docs say it should work... I will look into this - give me time

Reply:
I also don't really like these #if ...
I agree doc say it is supported, but when I go to definition from xcode, It doesn't say it is support on tvOS ...

Reply:
So I have double checked that and it's it as you say - strange 
Let's merge this